SabreTools.FileTypes/SabreTools.FileTypes.csproj

```diff
@@ -15,7 +15,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="SharpCompress" Version="0.29.0" />
+    <PackageReference Include="DiscUtils" Version="0.16.13" />
```

Comment:
I know nothing about this package, what does it do? Is there any way that the functionality can be replicated with SharpCompress or in self-written code?

Reply:
No, SharpCompress should stay. its complete different packages and is only for disc and hard disk type (chd, ntfs etc.). First idea is to read an iso (as archive or as whole file) and make an output as chd file. chd files also can be used for hard disk copies, so with discutils you can read every hard disk format and bring it into chd format. As code is designed its also possible to transform it into an iso file. At the moment Id only tested iso or chd packed in 7z files. They were also transformed expectedly. The tranform of chds back to iso or other formats does not work yet.